### ISSUE TITLE
Initial Components necessary to have

### ISSUE THREAD

**Issue by pushpendrahpx** [pushpendrahpx/conversion-lamp#3]
1. Navbar Component
2. Buttons
3. Landing Page Sections with ( 100hv specs )
4. File upload section

**Comment by rounit08** [comment]
@pushpendrahpx can I take this?

**Comment by pushpendrahpx** [comment]
Yeah sure, go with it, & as of now I know the design of product might not be clear, so for that we can have a discussion regarding it, so make sure you join slack channel & we'll discuss it there, is it fine?
